notebooks/124-hugging-face-hub/124-hugging-face-hub.ipynb

```diff
@@ -0,0 +1,463 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    ov_encoder_input = [v for v in dict(encoded_input).values()]
</pre><p>Is list necessary?</p><pre class="ql-syntax" spellcheck="false">ov_encoder_input = encoded_input.data
</pre><p>Using dict seems cleaner</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>I still need to cast </p><pre class="ql-syntax" spellcheck="false">dict(encoded_input)
</pre><p>but it looks cleaner.</p><p>Done</p>

Reply:
No need to create a new dict, `encoded_input.data` attribute is already a dict:
<img width="315" alt="image" src="https://github.com/openvinotoolkit/openvino_notebooks/assets/51917466/5be1f09c-1319-47ef-86dd-109a54223211">

Reply:
done